content/departments/cloud/working-agreements.md

```diff
@@ -1,8 +1,84 @@
-# Working agreements
+# Cloud Team Working Agreements
 
-Guidelines for how the [Cloud team](./index.md) collaborates. Also see [How we work](./index.md#how-we-work).
+Guidelines for how the [Cloud team](./index.md) collaborates.
 
-## Definition of Ready
+## Roles of Cloud teammates
+
+- [Cloud Engineer](#cloud-engineer)
+- [Cloud EM and PM](#cloud-em-and-pm)
+- [DRI](#dri)
+- [On-call Engineer](#on-call-engineer)
+- [Security Engineer](#security-engineer)
+
+## Issue tracking
+
+The [Cloud team GitHub Project](https://github.com/orgs/sourcegraph/projects/264/views/1) is the single source of truth.
+
+Learn more about [how we use GitHub Projects](./github-projects-beta.md)
+
+Learn more about [grooming and estimation process](./grooming-and-estimation-process.md)
+
+## Project planning
+
+### How do we work on projects?
+
+Aside from ad-hoc and support tasks, our work is usually considered as part of an initiative. This section will walk through the process of how to discover, plan, scope, estimate and prepare the work for execution. At a high level, every initiative is considered a project; every [project](#project-initiative) consists of multiple [milestones](#milestone); each milestone consists of many [tasks](#task).
+
+For example, below is the mental model to approach a project to add support for a new region to Cloud:
+
+> Adding new region support to Cloud is the project. Usually, the first milestone would be performing discovery work. In this case, this could be quickly putting a working deployment (proof of concept, or POC) in the new region. After the discovery, it provides a better idea of what needs to be done. The second milestone would be to properly define the scope and break them down to multiple milestones for execution. This could be presented in the form of a RFC or some other document and it should be peer reviewed by the team or relevant stakeholders. Such document should meet the requirement of [Definition of Ready](#definition-of-ready) prior to execution. The project should also meet the requirement of [Definition of Done](#definition-of-done) to be considered as finished.
```

Comment:
I'd recommend we use the term `milestone` for actual delivery milestones within the bigger initiative and for the project planning use the term `steps` with the first step being the discovery, the second step planning and scoping and planning and the third moving do delivery/execution.